locales/de-DE.json

```diff
@@ -2095,19 +2095,19 @@
     "verbal_multi_cue": {
       "phrases": {
         "0": "<CURRENT_VERBAL_CUE> Dann <NEXT_VERBAL_CUE>",
-        "1": "<CURRENT_VERBAL_CUE> Then, in <LENGTH>, <NEXT_VERBAL_CUE>"
+        "1": "<CURRENT_VERBAL_CUE> Dann, in<LENGTH>, <NEXT_VERBAL_CUE>"
```

Comment:
Verifying no space after `in`

Reply:
yeah thats a typo

Reply:
Fixed in https://github.com/valhalla/valhalla/pull/2432/commits/28a9e26385e525b108fbf3b1854b5864ffecff09.